### DISCUSSION TITLE
maintain file names and folders

### DISCUSSION THREAD

**Discussion by pino1536** [BoBoBaSs84/DDS.Tools#57] [Ideas]
Your tool works perfect ❤️ Actually I used it for an Oblivion to Unreal Engine Project.

But would be more helpful if it simple creates a 1:1 Result folder with same folder structure and file names.

Otherwise we need to write an extra tool for the Result.json to do it.

**Comment by BoBoBaSs84** [comment]
Sounds absolutely reasonable. I need to make some docs and while at, I will add some switches ( and document them too ;) that preserve the structure and file names.

@pino1536 don't tell me this is a oblivion & unreal engine 5 project? 🥹

**Reply by pino1536** [reply]
only for fun to learn some UE stuff... to recreate things like the UI and core elements witch the new UE Blueprint system.
